### DISCUSSION TITLE
Ekkko Station for Plug Data

### DISCUSSION THREAD

**Discussion by pereshaped** [plugdata-team/plugdata#1860] [General]
Hi all, just an update of Ekkko Station, which will be released this weekend. I have been working on the manual and can be read using the link below. I have only tested on a Mac, so I will be interested to see how it is on other platforms. There's some limiting aspects at the moment on iOS, which are outlined in the "Red Me First" section. Just finishing presets now and it's done.

[https://villezpere.wordpress.com/2024/09/18/ekkko-station-the-manual-main-in-progress/](url)

**Comment by timothyschoen** [comment]
The link gives me a 404, I'd like to try it :)

**Reply by dromer** [reply]
The link is wrong, here it is correct: [https://villezpere.wordpress.com/2024/09/18/ekkko-station-the-manual-main-in-progress/](https://villezpere.wordpress.com/2024/09/18/ekkko-station-the-manual-main-in-progress/)

I don't see any download on that page though ..

**Reply by pereshaped** [reply]
For some reason Github has wrapped the wordpress link with a link to github. At least you found the correct link.

**Comment by pereshaped** [comment]
The keywords are at the end of the title "In Progress"

Im launching it this weekend if all goes well.
